File: Lojban Numbers.py
key = {"no":0,"pa":1,"re":2,"ci":3,"vo":4,"mu":5,"xa":6,"ze":7,"bi":8,"so":9}
def convert_lojban(lojban):
    result = ""
    for i in range(0,len(lojban),2):
        result+=str(key[lojban[i:i+2]])
    return int(result)
    

File: Lojban Numbers.js
function convertLojban(lojban) {
  const dic = {"pa":1,"vo":4,"ze":7,"re":2,"mu":5,"bi":8,"no":0,"ci":3,"xa":6,"so":9}
  return parseInt(lojban.split("").map((_,i) => i%2 ===0 ? lojban.substring(i,i+2) : "").filter(v => v!=='').map(v => String(dic[v])).join(""))
}
